app/assets/stylesheets/modules/homepage.scss

```diff
@@ -0,0 +1,44 @@
+@import "variables";
+@import "mixins";
+
+body.home-page {
+  background: url(dog.png) no-repeat;
+  background-size: 100%;
+  opacity: .9;
+
+  .right {
+    padding: 1em;
+    padding-left: 2em;
+
+    a {
+      font-family: "Poiret One";
```

Comment:
Properties should be ordered border-bottom, color, font-family, font-size, margin-right, padding-bottom